packages/ckeditor5-html-support/tests/integrations/style.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import ClassicTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/classictesteditor';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import GeneralHtmlSupport from '../../src/generalhtmlsupport';
+import { getModelDataWithAttributes } from '../_utils/utils';
+import { getData as getModelData } from '@ckeditor/ckeditor5-engine/src/dev-utils/model';
+
+/* global console, document */
+
+describe( 'StyleElementSupport', () => {
+	const STYLE = 'div { color: red; }';
+	const CODE_CPP = 'cout << "Hello World" << endl;';
```

Comment:
Isn't it a leftover from the `<script>`'s tests?

Reply:
Fixed.